python/compiler/aiecc/main.py

```diff
@@ -465,7 +465,10 @@ async def process_core(
                 install_path, "aie_runtime_lib", aie_target.upper()
             )
 
-            clang_link_args = ["-Wl,--gc-sections"]
+            # --gc-sections to eliminate unneeded code.
+            # --orphan-handling=error to ensure that the linker script is as expected.
+            # If there are orphaned input sections, then they'd likely end up outside of the normal program memory.
+            clang_link_args = ["-Wl,--gc-sections", "--orphan-handling=error"]
```

Comment:
`"--orphan-handling=error"` threw an error locally for me: `error: unknown argument: '--orphan-handling=error'`

Reply:
Sorry, typo on my part